modules/TaskFormPresenter.js

```diff
@@ -0,0 +1,41 @@
+class TaskFormPresenter {
+  #input;
+
+  #submit;
+
+  #view;
```

Comment:
Please check if these declarations are needed. They are also created in the constructor.

Reply:
I'll try to remove these, but they are required for proper function.